pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -200,15 +225,7 @@ rec {
             "/bin/sh"
           ];
           __propagatedImpureHostDeps = computedPropagatedImpureHostDeps ++ __propagatedImpureHostDeps;
-        } // lib.optionalAttrs (outputs' != [ "out" ]) {
-          outputs = outputs';
-        } // lib.optionalAttrs (attrs ? doCheck) {
-          # TODO(@Ericson2314): Make unconditional / resolve #33599
```

Comment:
I think you lost this TODO comment when moving this code upwards: I don't think with this PR #33599 would be solved yet? As it would require at least turning `config.doCheckByDefault` to default to `true`, which isn't coming until at least hydra has performed a world-rebuild with it and checked it failed no additional package, I guess.

Reply:
Fair enough. Will fix.